docs/src/docs/reports/layout-check.adoc

```diff
@@ -0,0 +1,23 @@
+= Layout Checks
```

Comment:
Adding a link in <<#_layout_check, Layout Checks>> to the reporting would probably be good,